### ISSUE TITLE
Plotjuggler does not show correct values with ROS1

### ISSUE THREAD

**Issue by NR-Daichi** [facontidavide/PlotJuggler#904]
# Problem description
When plotjuggler is subscribing odometry topic with ROS1, the values in pose/pose/position and pose/pose/orientation are very huge values (> trillion) or 0
For the details, please watch the attached video

https://github.com/facontidavide/PlotJuggler/assets/141435269/5fa4b3e7-320c-4b27-909a-7c4697a64280

Joint_states topics are fine, Plotjuggler displays the right values

# Steps to reproduce (important)
Platform: Ubuntu 20.04, ROS1

Step 1:
install ros plugins by following the instructions written on this page ("How to compile PlotJuggler from source")
https://github.com/PlotJuggler/plotjuggler-ros-plugins

Step 2:
launch plotjuggler and subscribe odometry topic

If anyone knows the way to run plotjuggler with ros1 on ubuntu 20.04, please let me know

**Comment by lucasw** [comment]
Can you zip up a short rosbag of your odom topic and attach it here?

**Comment by peci1** [comment]
I get the same bug with apt-installed plotjuggler on Ubuntu 20.04/Noetic.

Sample odom topic values that get displayed as the nonsense:

```
x: 82.09575053694694
y: -8.41266412125995
z: -0.08068917743157186
```

**Comment by peci1** [comment]
Combination of PJ 3.8.5 from testing and PJ-ROS 1.7.0 from snapshot 2023-11-15 works.

**Comment by sunilsulania** [comment]
It started happening recently with me as well on ROS1 on ubuntu 20.04 apt installed version. I may have done an apt upgrade, I dont remember. 
Interestingly, Issue is with nav_msgs/Odometry position values. like mentioned in above image.

**Comment by NR-Daichi** [comment]
> Can you zip up a short rosbag of your odom topic and attach it here?

[ros_bagfiles.zip](https://github.com/facontidavide/PlotJuggler/files/13871499/ros_bagfiles.zip)
Please have a look at the odometry topic (ex. /right_foot_BF_pin/pose/pose/position/x ) 
I could plot them with rqt_plot
![Kazam_screenshot_00026](https://github.com/facontidavide/PlotJuggler/assets/141435269/6b241ec0-b0fe-4a68-8536-c3be9d337c2e)

**Comment by NR-Daichi** [comment]
> It started happening recently with me as well on ROS1 on ubuntu 20.04 apt installed version. I may have done an apt upgrade, I dont remember. Interestingly, Issue is with nav_msgs/Odometry position values. like mentioned in above image.

This suddenly happen to me as well. In my case, the Odometry/Twist / Twist / linear is also not correct

**Comment by facontidavide** [comment]
Fixed: 
![image](https://github.com/facontidavide/PlotJuggler/assets/2822888/66b34729-9009-450c-a115-714571d780d0)

**Comment by sunilsulania** [comment]
Thank you Davide.

**Comment by Veng97** [comment]
Can confirm this issue also happens in ROS2.

![Screenshot from 2024-01-13 15-32-26](https://github.com/facontidavide/PlotJuggler/assets/40239379/36fa54f1-b04e-419f-ba11-e895f3892792)
![Screenshot from 2024-01-13 15-32-41](https://github.com/facontidavide/PlotJuggler/assets/40239379/9a396e54-ef22-4e07-9c0f-a03e1e48b3ec)

The values shown in plotjuggler does not match the topic echo.

**Comment by facontidavide** [comment]
Problem fixed in 3.8.6

Unless you are using that version, don't duplicate this issue
